blocks/form/form.js

```diff
@@ -223,9 +225,11 @@ const createSelect = withFieldWrapper((fd) => {
     ph.setAttribute('disabled', '');
     select.append(ph);
   }
-  fd.Options.split(',').forEach((o) => {
+  const options = fd.Options.split(',');
+  const optionsName = fd['Options Name'] ? fd['Options Name'].split(',') : options;
+  options.forEach((o, index) => {
     const option = document.createElement('option');
-    option.textContent = o.trim();
+    option.textContent = optionsName[index].trim();
```

Comment:
We are still in progress of deciding how to move forward with options list. Country options contain ',' which we used to use as a delimiter.

Reply:
Lets processed with approach #1 i.e. endpoint